File: Frontend/sakai-ng-master/src/app/biblioteca/interfaces/OcurrenciaMaterialDTO.ts
// src/app/interfaces/ocurrencia-material-dto.ts
export interface OcurrenciaMaterialDTO {
  idMaterial:    number | null;
  codigoEquipo:  string;
  nombreEquipo:  string;
  cantidad:      number;
  costo:         number | null;
  esBiblioteca:  boolean | null;
}

File: Backend/login-microsoft365/src/main/java/com/miapp/model/dto/OcurrenciaMaterialDTO.java
package com.miapp.model.dto;

import java.math.BigDecimal;

public record OcurrenciaMaterialDTO(
        Long        idMaterial,
        String      codigoEquipo,
        String      nombreEquipo,
        Integer     cantidad,
        BigDecimal  costo,
        Boolean     esBiblioteca
) {}